### PR TITLE
Improve overall coding quality based on TA feedback

### PR DESCRIPTION
**Pull request by deltango**

### CODE REVIEW

File: src/main/java/utask/commons/util/UpdateUtil.java [hunk 1]
```diff
@@ -78,23 +82,22 @@ public static Task createEditedTask(ReadOnlyTask taskToEdit, Boolean value)
         case FLOATING:
             placeholder = new FloatingTask(taskToEdit.getName(),
```
Comment (yong24s) [inline src/main/java/utask/commons/util/UpdateUtil.java]: If you often create different instances like this.  Consider group these factory-like methods to a helper to generate instances.

File: src/main/java/utask/logic/commands/DoneCommand.java [hunk 2]
```diff
@@ -28,7 +32,7 @@
     public static final String MESSAGE_DONE_TASK_SUCCESS = "Done task: %1$s";
```
Comment (yong24s) [inline src/main/java/utask/logic/commands/DoneCommand.java:32]: Consider grouping constant messages into Message class

File: src/main/java/utask/logic/commands/UndoneCommand.java [hunk 3]
```diff
@@ -10,11 +13,12 @@
 import utask.model.task.ReadOnlyTask;
 import utask.model.task.Task;
 
+//@@author A0138423J
 /**
  * Edits the details of an existing task in the uTask.
  */
-// @@author A0138423J
 public class UndoneCommand extends Command implements ReversibleCommand {
+    private final Logger logger = LogsCenter.getLogger(UpdateCommand.class);
```
Comment (yong24s) [inline src/main/java/utask/logic/commands/UndoneCommand.java]: Wrong class
Supposed to be UndoneCommand.class

File: src/main/java/utask/logic/commands/UpdateCommand.java [hunk 4]
```diff
@@ -20,6 +22,7 @@
 
 // @@author A0138423J
 public class UpdateCommand extends Command implements ReversibleCommand {
+    private final Logger logger = LogsCenter.getLogger(UpdateCommand.class);
```
Comment (yong24s) [inline src/main/java/utask/logic/commands/UpdateCommand.java:25]: Same problem

**Review by yong24s — APPROVED** [review]
Good work. 

Please view my comments.

**Review by LIUJIAHAOCS — APPROVED** [review]
Good job on refactor isComplete to Status, the naming now makes more sense. Good work

**Comment by deltango** [comment]
#120 Resolved

**Pull request merged**